Basic/ClientDriven/Assets/Scripts/ServerPlayerMove.cs

```diff
@@ -9,17 +9,12 @@
 [DefaultExecutionOrder(0)] // before client component
 public class ServerPlayerMove : NetworkBehaviour
 {
-    private ClientPlayerMove m_Client;
-
     [SerializeField]
-    private Camera m_Camera;
+    ClientPlayerMove m_ClientPlayerMove;
 
     public NetworkVariable<bool> ObjPickedUp = new NetworkVariable<bool>();
-
-    private void Awake()
-    {
-        m_Client = GetComponent<ClientPlayerMove>();
-    }
+    
+    NetworkObject m_PickedUpObj;
```

Comment:
as per our [coding conventions](https://github.cds.internal.unity3d.com/unity/unity-meta/blob/stable/ReferenceSource/CSharp/Assets/CSharpReference.cs), we should not use abbreviations in code, unless we're dealing with acronyms